### PR TITLE
Ma/matchtabs in standard layout

### PR DESCRIPTION
**Pull request by nitro-marky**
## What does this change?

Matchtabs were not showing on test pages in DCR and this is potentially because they weren't included in the StandardLayout file. This adds them to it.

### After

Screenshot 2021-11-23 at 09 35 46

Tested locally.

### CODE REVIEW

**Review by rcrphillips — APPROVED** [review]
LGTM

**Review by lucymonie — CHANGES_REQUESTED** [review]
The change looks reasonable and straightforward but there seem to be some knock-on effects to some parts of the liveblog, which you can see in the chromatic snapshots. It would be worth digging into these changes.

**Review by oliverlloyd — COMMENTED** [review]
🤔 it's quirky that this placeholder div is appearing in the dom when the `matchtabs` grid area item isn't added to any grid

I think semantically we would want it to be in the grid css

**Comment by oliverlloyd** [comment]
> The change looks reasonable and straightforward but there seem to be some knock-on effects to some parts of the liveblog, which you can see in the chromatic snapshots. It would be worth digging into these changes.

I think a lot of these diffs are from the leftcol PR but _this one_ isn't and is a problem with this code
Screenshot 2021-11-23 at 10 22 16

This is where the GridItem is being dropped at the bottom of the page because it hasn't been told where to go

**Comment by nitro-marky** [comment]
So what would you recommend?

**Review by oliverlloyd — APPROVED** [review]

**Review by OllysCoding — APPROVED** [review]
Looks good!

File: dotcom-rendering/src/web/layouts/StandardLayout.tsx [hunk 1]
```diff
@@ -485,6 +490,18 @@ export const StandardLayout = ({ CAPI, NAV, format, palette }: Props) => {
...
+								{format.design === ArticleDesign.DeadBlog ||
+									(format.design === ArticleDesign.LiveBlog &&
+										CAPI.matchUrl && (
+											<Placeholder
+												rootId="match-tabs"
+												height={40}
+											/>
+										))}
+							</div>
+						</GridItem>
```
Comment (oliverlloyd) [inline dotcom-rendering/src/web/layouts/StandardLayout.tsx:504]:
```suggestion
						<GridItem area="matchtabs" element="aside">
							<div css={maxWidth}>
								{CAPI.matchUrl && (
									<Placeholder
										rootId="match-tabs"
										height={40}
									/>
								)}
							</div>
						</GridItem>
```

Comment (oliverlloyd) [reply dotcom-rendering/src/web/layouts/StandardLayout.tsx]: I missed this but we don't need this format check here as we're never going to be a live blog at this point, not in StandardLayout

**Comment by nitro-marky** [comment]
[Fix match tabs to show in articles again](https://trello.com/c/hY6MVUef/433-fix-match-tabs-to-show-in-articles-again)

**Review by lucymonie — APPROVED** [review]
👍🏻

**Pull request merged**